landing-pages/site/content/en/use-cases/plarium-krasnodar.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Plarium Krasnodar"
+linkTitle: "Plarium Krasnodar"
+quote:
+    text: "Apache Airflow helps us efficiently tackle crucial game dev tasks, such as working with churn or sorting bank offers."
+    author: "David Manukian"
+logo: "plarium-krasnodar-logo.svg"
+---
+
+##### What was the problem?
+Our Research & Development department carries out experiments that are aimed at finding better solutions. Previously, we didn't have any suitable tools with a sufficient number of built-in functions, and we had to orchestrate processes manually and entirely from scratch every time. This led to difficulties with dependencies and monitoring when building complex workflows. We needed a tool that would provide a more centralized approach so that we could see all the logs, the number of retries, and the task performance time. The most important thing that we lacked was the ability to backfill historical data and restart failed tasks.
```

Comment:
"...aimed at finding better solutions." <- this first sentence may be stronger if you expand on this point by giving either some general types of solutions or a specific one. For example (I'm totally making these up, they may have nothing to do with your use case 😄 ) 
- "better solutions to our customer's data analytics problems"
- "better solutions to slow, manual data pipelines"
- "better solutions to solve the problems our developers encounter daily."
- "alternatives to ..." (instead of solutions)

Reply:
@leahecole Thank you. We changed the text.